src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -354,6 +361,28 @@ public function testFilesExists()
         $this->assertTrue($this->filesystem->exists($basePath.'folder'));
     }
 
+    /**
+     * @expectedException \Symfony\Component\Filesystem\Exception\IOException
+     */
+    public function testFilesExistsFails()
+    {
+        if ('\\' !== DIRECTORY_SEPARATOR) {
+            $this->markTestSkipped('Test covers edge case on Windows only.');
+        }
+
+        $basePath = $this->workspace.DIRECTORY_SEPARATOR.'directory'.DIRECTORY_SEPARATOR;
```

Comment:
let's use `/` or `\` here instead of the constant, as we already know it's only windows and windows support both.